pennylane/qinfo/entropies.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Differentiable quantum entropies"""
+
+import pennylane as qml
+
+
+def vn_entropy_transform(qnode, indices, base=None):
```

Comment:
I know we want to distinguish between the related functions for measurement processes, numerical calculation, and qnode transform, but would it be enough to rely on the distinction by submodule ( `qml.` vs `qml.math.` vs `qml.qinfo.` respectively), because we've yet to name any qnode transform "transform" (see grad, jacobian, param_shift, metric_tensor, etc).

Reply:
@co9olguy What do you think about the different names? Could we have same name in different module? It is kind of redundant to call it `vn_entropy_transform`, I totally agree.

Reply:
@rmoyard the namespace can definitely be used to distinguish functions, they don't have to all have distinct names :)